spec/conformance/index.md

```diff
@@ -24,9 +24,39 @@ The server SHALL convey the *optional* SMART Core Capabilities it supports using
 
 - A [Well-Known Uniform Resource Identifiers (URIs)](#using-well-known) JSON file.
 
-### Core Capabilities
+### Capability Sets
+
+A *Capability Set* combines individual capabilities to enable a specific use-case. A SMART on FHIR server SHALL support one or more *Capability Set*s. Unless otherwise noted, each capability listed is required to satisfy a *Capability Set*. Any individual SMART server will publish a granular list of its capabilities; from this list a client can determine which of these Capability Sets are supported:
```

Comment:
```suggestion
A *Capability Set* combines individual capabilities to enable a specific use-case. A SMART on FHIR server SHOULD support one or more *Capability Set*s. Unless otherwise noted, each capability listed is required to satisfy a *Capability Set*. Any individual SMART server will publish a granular list of its capabilities; from this list a client can determine which of these Capability Sets are supported:
```